uefi-hello-world-aarch64/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Python cache files.
+/__pycache__
+
+# OVMF images, which provide a UEFI firmware for QEMU.
+/OVMF_CODE.fd
+/OVMF_VARS.fd
```

Comment:
These should be updated to be the `*-pflash.raw` files which are used by the script.